src/components/IllustrationItem/IlloImage.jsx

```diff
@@ -0,0 +1,68 @@
+import cls from 'classnames';
+import {
+  ArrowTopRightOnSquareIcon,
+  DocumentIcon,
+} from '@heroicons/react/24/solid';
+import { colors, flex } from '../../theme';
+import styles from './styles.module.css';
+
+export default function IlloImage({
+  url,
+  slug,
+  onClick,
+  hoverState,
+  setHoverState,
+  isArchive,
+}) {
+  const imgContainer = cls(styles.imgContainer, flex.flex, flex.flexCenter, {
+    [colors.bgSlate700]: !isArchive,
+  });
+
+  if (!url) {
```

Comment:
Hmm this component is starting to be a little unwieldy. What do you think about having two components, IlloImage and IlloNoImage, and then both take a IlloHover as a child to avoid duplicating that JSX?

Reply:
I'm open to other ideas, but just feels like this component needs to be taken apart a bit more.

Reply:
It can actually get even more DRYer than that since the only real difference between the two is an `<img>` or Icon depending on a valid URL. Let me know what you think.